package.json

```diff
@@ -178,5 +178,10 @@
     "transform-loader": "^0.2.4",
     "true-case-path": "^2.2.1",
     "virtual-webgl": "^1.0.6"
+  },
+  "overrides": {
+    "falafel": {
+      "acorn": "^8.1.1"
```

Comment:
I don't feel comfortable introducing overriding dependencies like that in this PR, specially if we want to release a patch.
IMHO This should be discussed in a separate PR.
I suggest we use bring back the if statement.